app/MMB/src/navigation/NavigationStack.js

```diff
@@ -78,21 +49,36 @@ const MainStack = StackNavigator(
   },
 );
 
+const TravelDocumentModalStack = StackNavigator(
+  {
+    TravelDocumentModalScreen: {
+      screen: withMappedProps(TravelDocumentModalScreen),
+    },
+  },
+  {
+    ...StackNavigatorOptions,
+    initialRouteName: 'TravelDocumentModalScreen',
+  },
+);
+
 export default StackNavigator(
   {
     MMBMainStack: {
       screen: MainStack,
     },
-    TravelDocumentScreen: {
-      screen: TravelDocumentStack,
+    MyModalStack: {
```

Comment:
Is the name final?

Reply:
Yeah, I was a little bit to fast here. Let me rename that